servers/shareable-lists-api/src/database/mutations/ShareableList.ts

```diff
@@ -173,6 +173,7 @@ export async function createAndAddToShareableList(
   const completeList: ShareableList = {
     ...list,
     // bigint causes issues again...
+    id: list.id as unknown as bigint,
```

Comment:
🤣

Reply:
I know... should I be better? yes. but is a number in javascript going to exceed bigint realistically?